src/components/KMultiselect/KMultiselect.vue

```diff
@@ -173,47 +173,49 @@
                   @update:model-value="onQueryChange"
                 />
               </div>
-              <KMultiselectItems
-                :items="sortedItems"
-                @selected="handleItemSelect"
-              >
-                <template #content="{ item }">
-                  <slot
-                    class="multiselect-item"
-                    :item="item"
-                    name="item-template"
-                  />
-                </template>
-              </KMultiselectItems>
-              <KMultiselectItem
-                v-if="enableItemCreation && uniqueFilterStr && !$slots.empty"
-                key="multiselect-add-item"
-                class="multiselect-add-item"
-                data-testid="multiselect-add-item"
-                :item="{ label: `${filterString} (Add new value)`, value: 'add_item' }"
-                @selected="handleAddItem"
+              <div aria-live="polite">
```

Comment:
Note for PR review: the change here is adding a wrapper div with aria-live attribute